src/app/pages/jobs/job-modal/job-modal.component.html

```diff
@@ -40,12 +40,10 @@ <h4 class='title areas'>Skills</h4>
                             </mat-chip>
                         </mat-chip-list>
                     </div>
-                    <div *ngIf='job.filled' id='filledprofile'>
-                        <h4 class='title areas'>Filled By</h4>
-                        <br>
-                        {{ job.profile.firstName }} {{ job.profile.lastName }}
+                    <div *ngIf='job.isFilled' id='filledprofile'>
```

Comment:
Why was this change made?

Reply:
id job.profile did not exist on the backend. There was no use for this particular piece of code. Pointed to a null variable, which caused compilation errors.